src/datasource/components/ConfigEditor.tsx

```diff
@@ -79,7 +80,22 @@ export class ConfigEditor extends PureComponent<Props, State> {
 
         <FieldSet label={'TwinMaker settings'} data-testid="twinmaker-settings">
           <InlineFieldRow>
-            <InlineField label="Workspace" labelWidth={16}>
+            <InlineField
```

Comment:
Our UX wants to move the ARN below the workspace since workspace is mandatory and the write ARN isn't. We'd also like it a bit clearer that the write ARN is optional by having a check or toggle that asks if their using Alarm Configuration and expands to show the Write ARN field if they want it.

Reply:
moved below workspace in https://github.com/grafana/grafana-iot-twinmaker-app/pull/117/commits/985697214f72e1fcbf371373f00e9473e9d8b0af